ql/patterns/observable.hpp

```diff
@@ -119,13 +119,17 @@ namespace QuantLib {
         virtual ~Observer();
 
         // observer interface
-        std::pair<iterator, bool>
+        virtual std::pair<iterator, bool>
         registerWith(const ext::shared_ptr<Observable>&);
 
         /*! register with all observables of a given observer. Note
             that this does not include registering with the observer
-            itself. */
-        void registerWithObservables(const ext::shared_ptr<Observer>&);
+            itself.
+
+            \deprecated consider using LazyObject::alwaysForwardNotifications() instead,
```

Comment:
The comment is out of date now, I guess?

Reply:
It is, yes.